cvmfs/network/jobinfo.h

```diff
@@ -185,13 +188,14 @@ class JobInfo {
   InterruptCue *interrupt_cue() const { return interrupt_cue_; }
   cvmfs::Sink *sink() const { return sink_; }
   const shash::Any *expected_hash() const { return expected_hash_; }
-  const std::string *extra_info() const { return extra_info_; }
+  const std::string *path_info() const { return path_info_; }
 
   off_t range_offset() const { return range_offset_; }
   off_t range_size() const { return range_size_; }
 
   CURL *curl_handle() const { return curl_handle_; }
   curl_slist *headers() const { return headers_; }
+  const std::string info_header_contents(const std::string templ);
```

Comment:
That's more than just a getter, and shouldn't be `const`, so I'd suggest
```suggestion
  std::string GetInfoHeaderContents(const std::string &templ);
```